main.go

```diff
@@ -115,6 +116,21 @@ func getCerts(file, format string) ([]*x509.Certificate, error) {
 				certs = append(certs, cert)
 			}
 		}
+	case "JKS":
+		scanner := bufio.NewReader(os.Stdin)
+		fmt.Print("Enter password: ")
+		password, _ := scanner.ReadString('\n')
+		keyStore, err := jceks.Load(file, []byte(strings.TrimSuffix(password, "\n")))
+		if err != nil {
+			return nil, err
+		}
+		for _, certName := range keyStore.ListCerts() {
```

Comment:
You should also iterate over private key entries, as they have certificates associated with them.